unity-renderer/Assets/Scripts/MainScripts/DCL/Components/TextShape/TextShape.cs

```diff
@@ -14,6 +14,14 @@ public class TextShape : BaseComponent
         [Serializable]
         public class Model : BaseModel
         {
+
+            //These values exist in the SDK but we are doing nothing with these values
+            public string fontWeight = "normal";
+            public bool resizeToFit = false;
+            public float zIndex = 0;
+            public bool isPickable = false;
```

Comment:
Shall we remove them from the SDK?

Reply:
Yeah, I think we should remove them even if we want to implement them in the future.